pkg/splunk/enterprise/finalizers_test.go

```diff
@@ -37,6 +37,8 @@ func splunkDeletionTester(t *testing.T, cr splcommon.MetaObject, delete func(spl
 		component = "standalone"
 	case "LicenseMaster":
 		component = "license-master"
+	case "MonitoringConsole":
```

Comment:
Does Splunk Monitoring Console deployment automatically gets deleted if we delete, lets say, a SHC which also deployed MC with it?

Reply:
This is a mistake, we actually don't need this case here. And no, irrespective of which CR creates the deployment it will not get deleted on the deletion of CR as we are not setting it as the owner of the MC pod

Reply:
Ok, I think we should set the CR as owner of MC pod, just like we do it in the case of SHC where we create a deployer too with SHC members. But the deployer is also deleted when we delete SHC. From user experience, it is not a good experience if while deploying a CR we create an MC pod too but while deleting we have to delete the MC pod explicitly.

Reply:
The idea for MC is to be always present. Any CR that comes up first just deploys the MC pod, after that as you apply new CR it just adds itself to the existing MC pod. Like deployer service is only attached to that specific shc, MC is for the entire namespace. In that way, even if cr gets deleted MC still needs to be running for that namespace. If this might create confusion among customers we can mention this flow in the release notes